File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/community/CommunityUiState.kt
package com.hhp227.concafe.presentation.community

import com.hhp227.concafe.domain.model.CommunityPost

data class CommunityUiState(
    val isLoading: Boolean = false,
    val posts: List<CommunityPost> = emptyList(),
    val nextCursor: String? = null,
    val hasNext: Boolean = false,
    val isLoadingMore: Boolean = false,
    val errorMessage: String? = null
)

File: iosApp/iosApp/Presentation/Main/Community/CommunityUiState.swift
//
//  CommunityUiState.swift
//  ConCafe
//
//  Created by 홍희표 on 4/27/26.
//

import Foundation
import Shared

struct CommunityUiState {
    var isLoading: Bool = false
    var posts: [CommunityPost] = []
    var nextCursor: String? = nil
    var hasNext: Bool = false
    var isLoadingMore: Bool = false
    var errorMessage: String? = nil
    var nativeAds: [Int32: any NativeAdHandle] = [:]
}
